lib/rubocop/target_ruby.rb

```diff
@@ -141,9 +141,17 @@ def find_version
           return versions.compact.min
         end
 
+        return gem_requirement_version(version) if version.send_type?
+
         version_from_str(version.str_content)
       end
 
+      def gem_requirement_version(version)
```

Comment:
As this is still extracted from the gemspec it should probably be part of that existing method.

Reply:
Initially I had this logic on the existing method (`find_version`), but I got the _Assignment Branch Condition_ and the _Cyclomatic complexity_ too high, that's why I created a new method.

Any suggestions @bbatsov?

Reply:
New arrangement based on https://github.com/rubocop-hq/rubocop/pull/9037#discussion_r523170441, let me know your thoughts @bbatsov .